page_distances.py

```diff
@@ -47,6 +47,7 @@ def show_page():
         st.write('Angular diameter distance at redshift', z_value, 'is:', str(round(bg.angular_diameter_distance(float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value) ), sig_digits)), 'Mpc')
         st.write('Comoving volume at redshift', z_value, 'is:', str(round(1e-9*bg.comoving_volume(float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value) ), sig_digits)), 'Gpc³')
         st.write('Lookback time at redshift', z_value, 'is:', str(round(bg.lookback_time(float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value) ), sig_digits)), 'Gyrs')
+        st.write('An object spanning an angle of 1 arcsec at redshift', z_value, 'is', str(round(bg.proper_separation(180/3600/np.pi, float(z_value), H0=float(H0_value), ΩM=float(ΩM_value), ΩDE=float(ΩDE_value), ΩR=float(ΩR_value), w0=float(w0_value) , wa=float(wa_value)), sig_digits)), 'kpc across.')
```

Comment:
Maybe this comment is more general than the scope of this PR, but these sentences should be ordered in a better way.
For instance, I would move the new sentence just after the one about the angular diameter distance.

Reply:
That makes sense, but I'd change that after the merge. What's important here is that all the information is correct, I think.

Reply:
On second thought, let me change it now.